engine/html5_web_api.cpp

```diff
@@ -41,6 +41,55 @@ void bind_api_web_view(CefRefPtr<CefV8Value> stingray_ns)
 		browser::pick(mouse_pos, from, ray);
 		return CefV8Value::CreateBool(true);
 	});
+
+	bind_api(ns, "create", [](const CefV8ValueList& args)
+    {
```

Comment:
aggh... ok. I thought I set my editor to tabs but I'll double check.  I have a couple projects where I need to switch back and forth and maybe i didn't catch it here.